t/10-cache.t

```diff
@@ -87,6 +87,8 @@ for my $type (qw/warehouse mlwarehouse/) {
   is( scalar @{$clims}, 2, 'two lims objects returned');
   is( $clims->[0]->driver_type, 'ml_warehouse', 'correct driver type');
 
+  SKIP: {
```

Comment:
We should only skip if we cannot find the relevant old warehouse module - we've not purged it from our deployments yet. So some ```eval require``` in the test of whether to skip or not perhaps?